site/pages.json

```diff
@@ -441,18 +417,7 @@
       "title": "schwimmen",
```

Comment:
And here

Reply:
With those change requests I can work with

Reply:
changed to "Triathlon"